vimeo-networking/src/test/java/com/vimeo/networking/model/appconfiguration/AppConfigurationTest.java

```diff
@@ -0,0 +1,18 @@
+package com.vimeo.networking.model.appconfiguration;
+
+import com.vimeo.networking.Utils;
+
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link AppConfiguration}.
+ * <p>
+ * Created by restainoa on 4/20/17.
+ */
+public class AppConfigurationTest {
+
+    @Test
+    public void verifyTypeAdapterWasNotGenerated() throws Exception {
+        Utils.verifyNoTypeAdapterGeneration(AppConfiguration.class);
```

Comment:
A while ago, I went through to see what could be added and what shouldn't to stag and learned that basically all the data returned from the `/configs` endpoint is broken https://vimean.atlassian.net/browse/ADF-100. The only reason we didn't notice it until that ticket is that we aren't actually using a lot of the configuration stuff in the client app. Because the API endpoint is busted, I opted to not migrate those classes to stag. It's been low priority since we don't really care about the configs endpoint (I can up the priority if you feel it needs to change).